app/views/dashboards/_dashboard_article_row.html.erb

```diff
@@ -112,6 +108,9 @@
       </button>
 
       <div class="crayons-dropdown top-100 right-0 z-10 align-left js-dashboard-row-more-dropdown">
+        <% if @current_user_pro %>
```

Comment:
Is pro coming back?!

Reply:
nope, Ben just moved it around, we use pro only for stats now